app/domain/papers/repositories/papers.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module PaperDeep
+  module Repository
+    # Repository for Members
+    class Papers
+      def self.all
+        Database::PaperOrm.all.map { |db_project| rebuild_entity(db_project) }
+      end
+
+      def self.find_eid(eid)
+        rebuild_entity Database::PaperOrm.first(eid: eid)
+      end
+
+      def self.find_publication_by_id(publication_id)
+        # SELECT * FROM `papers`
+        # LEFT JOIN `publications` ON (`publications`.`pid` = `papers`.`publication_id`)
+        # WHERE (`publication_id` = publication_id)
```

Comment:
You might want to keep some function in code with commented-out.
Here is another way to do. Put the function in the `Wiki`.
The link is [another team's review](https://github.com/SOA-nuts/PortfolioAdvisor/pull/15) and example of the [`Wiki` useage](https://github.com/SOA-nuts/PortfolioAdvisor/wiki).